apps/app/src/views/components/OrderStatusBadge.tsx

```diff
@@ -69,9 +72,27 @@ export const ORDER_FULFILLMENT_TIP_MAP: OrderFulfillmentRecord = {
   DELIVERED: 'Mail order delivered'
 };
 
+export const ORDER_STATE_TIP_MAP: { [key in types.OrderState]: string } = {
+  PLACED: 'Order has been placed',
+  ROUTING: 'Order waiting on patient pharmacy selection',
+  PENDING: 'Order is pending',
+  CANCELED: 'Order has been canceled',
+  COMPLETED: 'Order has been completed',
+  ERROR: 'Order has an error'
+};
+
+export const ORDER_STATE_COLOR_MAP: { [key in types.OrderState]: string } = {
```

Comment:
will these colors match the order page as well? I think canceled used to be gray, just want to check that they'll match